types/webpack-node-externals/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for webpack-node-externals 2.5
+// Type definitions for webpack-node-externals 3.0
```

Comment:
please bump minimum TS version to 4.4 (L.7):
```ts
// TypeScript Version: 4.4
```
this should fix the CI errors

Reply:
Thanks for the tip, I tried a few versions locally to no avail. TS4.3 & up produce errors about `buffer` types being required but not part of the dependencies. Should `@types/node` be added as a peer?

Reply:
I’ve gone in and implemented the same “fix” as https://github.com/DefinitelyTyped/DefinitelyTyped/pull/64081#pullrequestreview-1269496629